db/seeds.rb

```diff
@@ -709,6 +709,7 @@ def file2.original_filename
 
   11.times do |_i|
     Digitization::Book.create(peel_id: rand(1..3400), part_number: rand(1..100),
```

Comment:
Minor Nit: Since we create the book here then right after doing a `.tap` then `save!` inside, we essentially doing two DB trips to the DB. 

Think if you just did a `.new` here instead of `.create` then your only saving the object once (in the `.tap`)

Needs testing though 😄 
```suggestion
    Digitization::Book.new(peel_id: rand(1..3400), part_number: rand(1..100),
```